uiowa/uids
# 4.x button updates

Resolves #706
Resolves https://github.com/uiowa/uids/issues/710

## To test
- check to see that the transparent option is present, and that the transparent borderless option has link underlining.
- test that going to font awesome and copying an icon ([An example icon is here, copy everything in the `<i>` tag](https://fontawesome.com/v5/icons/dog?s=solid))
- Test that the full width option is available and working properly
- check that the borderless option works for both the card and button
- Test background color combinations, if possible https://github.com/uiowa/uids/pull/716#issuecomment-1170507596
